lib/loader.js

```diff
@@ -155,6 +155,8 @@ module.exports = function (content) {
           return loader + '!' + rewriter + ensureBang(ensureLoader(lang))
         case 'script':
           return injectString + ensureBang(ensureLoader(lang))
+        default:
+          return ensureBang(loaders[type])
```

Comment:
Sorry, should have replied to this.  This won't throw because there's a guard on line 289 that emits a warning in this case.  This is covered by the `warning should be raised when a loader is not specified for a custom block` unit test.